database/deepdive-load

```diff
@@ -1,14 +1,24 @@
 #!/usr/bin/env bash
 # deepdive-load -- Loads a given relation's data
-# > deepdive load RELATION [SOURCE...]
-# Initializes given RELATION in the database and loads data from SOURCE.
-# When SOURCE is unspecified, the data is loaded from files found on path
-# input/RELATION.* under the DeepDive application.
+# > deepdive load RELATION[(COLUMN[,COLUMN]...)] [SOURCE...]
+# Initializes given RELATION in the database and loads data from SOURCE for
+# optionally specified COLUMNs.  When SOURCE is unspecified, the data is loaded
+# from files found on path input/RELATION.* under the DeepDive application.
+#
+# When RELATION is a random variable and no COLUMN is specified, the SOURCE are
+# expected to contain the `label` column at the end, in addition to all
+# user-defined ones.
```

Comment:
This requires the users to be aware of the `label` column in variable relation, but not declaring it in the schema declaration. IMO it would be better to be more consistent: either the `label` column is explicitly exposed to the users, or it's completely hidden from the users.